src/styles/global.scss

```diff
@@ -40,3 +40,14 @@ textarea {
 a {
   color: var(--color-text-default);
 }
+
+// these spacer utilities are used to add hard coded spaces to components
+// names of these spacer classes is coming from the api.
+// One sample usage is for Indopak words, some words need extra space at the end(see 2:2:4)
+// detecting these spaces in FE is hard, and will slow down the app
```

Comment:
I am not in favor of having this as a global class since it's only used inside `QuranText`. what I suggest doing is moving those 2 classes to `TextWord.module.scss` and to use them, you can do something like this inside `TextWord`:

`[styles[cssClass]]: cssClass` like the other existingcases with using different fonts